src-vue/src/components/PullRequestsSelector.vue

```diff
@@ -30,17 +24,15 @@
                     </el-alert>
                 </div>
                 <p v-if="pull_requests_mods.length === 0">
-                    <el-progress
-                        :show-text="false"
-                        :percentage="100"
-                        status="warning"
-                        :indeterminate="true"
-                        :duration="1"
-                        style="margin: 15px"
-                    />
+                    <el-progress :show-text="false" :percentage="100" status="warning" :indeterminate="true" :duration="1"
+                        style="margin: 15px" />
                 </p>
                 <el-card v-else shadow="hover" v-for="pull_request in pull_requests_mods" v-bind:key="pull_request.url">
                     <el-button type="primary" @click="installModsPR(pull_request)">Install</el-button>
+                    <a target="_blank"
```

Comment:
Why do you put this button inside a link tag, whereas you only use a button to install launcher PRs?
This prevents applying the same style to both buttons.

Reply:
Cause that was the quickest way I found to implement the feature... I guess I should instead have the button call some TypeScript that opens the browser?

Reply:
Yeah, just like you did with `downloadLauncherPR`